src/libraries/Microsoft.PowerFx.Connectors/Execution/OpenApiJsonSerializer.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.PowerFx.Connectors.Execution
+{
+    internal class OpenApiJsonSerializer
+    {
+        private readonly Dictionary<string, object> _dic;
+        private readonly MemoryStream _stream;
+        private readonly Utf8JsonWriter _writer;
+        private readonly OpenApiSchema _schema;
+
+        internal OpenApiJsonSerializer(OpenApiSchema schema, Dictionary<string, object> dictionary)
+        {
+            _schema = schema;
+            _dic = dictionary;
+            _stream = new MemoryStream();
+            _writer = new Utf8JsonWriter(_stream, new JsonWriterOptions());
+        }
+
+        internal string ToJson()
+        {
+            _writer.WriteStartObject();
+            foreach (var prop in _schema.Properties)
+            {
+                WriteProperty(prop);
+            }
+
+            _writer.WriteEndObject();
```

Comment:
This case can be merged with the case "object" below. 
Ie like this: , [https://github.com/microsoft/PowerApps-Language-Tooling/blob/dae85de910ebff06e7a5b9b8e10c50b322e8400b/src/PAModel/Utility/JsonNormalizer.cs#L39](https://github.com/microsoft/PowerApps-Language-Tooling/blob/dae85de910ebff06e7a5b9b8e10c50b322e8400b/src/PAModel/Utility/JsonNormalizer.cs#L39)

Reply:
I don't understand